qutebrowser/browser/commands.py

```diff
@@ -643,14 +643,29 @@ def scroll_perc(self, perc: {'type': float}=None,
 
     @cmdutils.register(instance='command-dispatcher', hide=True,
                        scope='window', count='count')
-    def scroll_page(self, x: {'type': float}, y: {'type': float}, count=1):
+    def scroll_page(self, x: {'type': float}, y: {'type': float}, *,
+                    top_navigate: {'type': ('prev', 'decrement')}=None,
+                    bottom_navigate: {'type': ('next', 'increment')}=None,
+                    count=1):
         """Scroll the frame page-wise.
 
         Args:
             x: How many pages to scroll to the right.
             y: How many pages to scroll down.
+            bottom_navigate: :navigate to the next page on bottom
+            top_navigate: :navigate to the previous page on top
```

Comment:
I think this could be improved a bit, as it's unclear what the argument means. Though I'm not sure how... Maybe ":navigate action to execute when at the bottom" or so?